apps/condo/domains/resident/schema/FindOrganizationsForAddressService.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Generated by `createservice resident.FindOrganizationsForAddressService --type queries`
+ */
+
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/resident/access/FindOrganizationsForAddressService')
+const {
+    MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC,
+    RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC,
+} = require('@condo/domains/resident/constants/constants')
+const {
+    findBillingReceiptsForOrganizations,
+    getOrganizationsWithAcquiring,
+    getOrganizationsWithMeters,
+    getBillingInformationForOrganizations,
+} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsForAddress')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+const checkLimits = async (uniqueField) => {
+    await redisGuard.checkCustomLimitCounters(
+        `find-organizations-by-address-${uniqueField}`,
+        RESIDENT_FIND_ORGANIZATIONS_WINDOW_SEC,
+        MAX_RESIDENT_FIND_ORGANIZATIONS_BY_WINDOW_SEC,
+    )
+}
+
+const FindOrganizationsForAddressService = new GQLCustomSchema('FindOrganizationsForAddressService', {
+    types: [
+        {
+            access: true,
+            type: 'input FindOrganizationsForAddressInput { addressKey: String!, unitName: String, unitType: String, tin: String, accountNumber: String }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressOrganizationType { id: ID!, name: String!, tin: String!, type: OrganizationTypeType! }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressAccountType { number: String!, category: ID!, balance: String, routingNumber: String!, bankAccountNumber: String! }',
+        },
+        {
+            access: true,
+            type: 'type FindOrganizationForAddressOutput { organization: FindOrganizationForAddressOrganizationType!, account: FindOrganizationForAddressAccountType, hasMeters: Boolean!, hasBillingData: Boolean! }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canFindOrganizationsForAddress,
+            schema: 'findOrganizationsForAddress (data: FindOrganizationsForAddressInput!): [FindOrganizationForAddressOutput]',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { addressKey, unitName, unitType, tin, accountNumber } = data
+                if (context.authedItem.type === RESIDENT) {
+                    await checkLimits(context.authedItem.id)
+                }
+                const properties = await find('Property', { addressKey, deletedAt: null })
```

Comment:
I guess it's bad search, if multiple managing companies found, I should receive the receipt only from connected one, shouldn't I?

Reply:
You can make a discision on client side
There is a common case: EPS has receipts and Management Company has meters. They both has the same tin, but different organizations ids. There is a chance to show the same receipt to user from different sources and it will be solved in allResidentBillingReceipts